api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -702,10 +702,10 @@ public void testMe_verifiedInstitutionalAffiliation_invalidEmail() {
   }
 
   @Test
-  public void test_AcceptTermsOfService() {
+  public void test_acceptTermsOfServiceDeprecated() {
     createAccountAndDbUserWithAffiliation();
     profileController.acceptTermsOfService(1);
-    verify(mockFireCloudService).acceptTermsOfService();
+    verify(mockFireCloudService).acceptTermsOfServiceDeprecated();
```

Comment:
Do we need a test for testing user has not accepted TOSDeperecated until we move away from it

Reply:
Could you say a little more about what you mean?  There's no way we can explicitly say "I do not accept Terra ToS" or "I reject Terra Tos"

Instead I added some checks to the create-account tests to show the cases when we do call this method and when we don't.  Does that fit what you're looking for?

Reply:
Got it. Thanks.